src/test/org/codehaus/groovy/runtime/FileAppendTest.groovy

```diff
@@ -19,6 +19,7 @@
 package org.codehaus.groovy.runtime
 
 import groovy.test.GroovyTestCase
+import groovy.lang.DummyGStringBase
```

Comment:
I am not sure I like ever having to import something from groovy.lang or groovy.util.
We have IDEs and codenarc checks that are going to complain too.
I would expect just a slower path in those cases if not in a cache.

Reply:
Understood. 
In fact the classes are not real groovy-core classes, I call them fake core classes ;-)
I am not sure whether we support declaring arbitrary classes under groovy-core packages and importing by default. AFAIK, Java does not allow this kind of declaration...